htlcswitch/link.go

```diff
@@ -2558,7 +2558,8 @@ func (l *channelLink) Peer() lnpeer.Peer {
 // ChannelPoint returns the channel outpoint for the channel link.
 // NOTE: Part of the ChannelLink interface.
 func (l *channelLink) ChannelPoint() *wire.OutPoint {
-	return l.channel.ChannelPoint()
+	chanPoint := l.channel.ChannelPoint()
```

Comment:
maybe change the order of commits, so this change is not necessary because we return a value in the second last commit here as well ?

Reply:
I'd rather tell a clean story with each commit. I get that there's some redundancy but I tried to work from the inside out. If I work from the outside in I'd probably have similar issues. If I combine the commits it's just a bit big and I wanted to make review easy. Nobody wants to sign up for extra review when there isn't a good reason for it so I want these types of refactor commits to be so smooth that people find them to be a joy to review.